docs/design-documents/features/connectivity/TLSSocket - Design document.md

```diff
@@ -75,13 +106,13 @@ Aim is to first support only certificate based authentication, so we implement o
 
 ## High Level Design Goal 3: Support both blocking and non-blocking operations
 
-As the Mbed TLS is already work with both socket types, we are able to `TLSocketWrapper`
+As the Mbed TLS already work with both socket types, we are able to `TLSocketWrapper`
```

Comment:
I think this was meant to be "As the Mbed TLS already work**s** with both socket types, we are able to **create** `TLSocketWrapper` that can handle both types as well."
I am sorry for this nitpicking, but once we are doing some corrections, let's make them final ;-)

Reply:
No problem. Fixed.